frontend/src/components/HomePage/Middle/PostFeed.js

```diff
@@ -88,21 +88,28 @@ function PostFeed() {
                 <div key={post.id} className="individual-post">
                   <div className="post-header">
                     <Link to={`/ProfilePage/${post.user_id}`}>
-                      <img className="post-pic" src={simpleUsers[post.user_id].profile_picture ? simpleUsers[post.user_id].profile_picture : profilePic}></img>
+                      <img
+                        className="post-pic"
+                        src={
+                          simpleUsers[post.user_id].profile_picture
+                            ? simpleUsers[post.user_id].profile_picture
+                            : profilePic
+                        }
```

Comment:
Would be nice to keep this consistent with other places
```suggestion
                        src={ simpleUsers[post.user_id].profile_picture || profilePic}
```